[UK] Add Storyboards (#4719)
ppy/osu-wiki: wiki/Storyboards/uk.md

# Storyboards
 
![Ця сцена показується під час гри Walkie Talkie Man. Однак, вона показується тільки тоді, коли ви отримуєте "Elite Beat 300".](img/Sbpassing.png "This scene is shown while playing Walkie Talkie Man. However, it is only shown when you get an Elite Beat 300.")

> ![Це видно, коли ви пропускаєте або отримуєте 50 в раніше зігранному комбо.](img/Sbfailing.png "This is shown when you missed or got a 50 in the previously played combo.")

Reviewer: ```suggestion
![Це показано під час пропускання або отримування 50 в раніше зігранному комбо.](img/Sbfailing.png "This is shown when you missed or got a 50 in the previously played combo.")
```

> **Сторіборди** (СБ) — це спеціально створений анімований фон, який супроводжує [бітмапу](/wiki/Beatmaps), часто для декоративних, а іноді й для ігрових цілей. Вони можуть складатися майже з усього, але зазвичай складаються з візуальних ефектів, призначених зробити карту більш естетично привабливою та самобутньою. Ця ідея прийшла з гри [Osu! Tatakae! Ouendan](/wiki/Glossary#ouendan). Подібно натхненню при створенні сторібордів, вони можуть підлаштовуватися під продуктивність гравця при грі і відображати різні речі в залежності від того, наскільки добре гравець справляється.

Reviewer: ```suggestion
**Сторіборд** (СБ) — це спеціально створений анімований фон, який супроводжує [бітмапу](/wiki/Beatmaps), часто для декоративних або ігрових цілей. Вони можуть складатися майже з усього, але зазвичай містять візуальні ефекти, котрі призначені зробити карту більш естетично привабливою та самобутньою. Ця ідея прийшла з гри [Osu! Tatakae! Ouendan](/wiki/Glossary#ouendan). Залежачи від натхнення, сторіборд може підлаштовуватись під продуктивність користувача при грі і відображати різні речі в залежності від того, наскільки добре гравець справляється.
```

Сторіборди зберігаються в папках бітмапи як окремі [`.osb` файли](/wiki/osu!_File_Formats/Osb_(file_format)) або розширення `[Events]` в розділі [`.osu` файлу](/wiki/osu!_File_Formats/Osu_(file_format)). Завдяки цьому можна створювати різні розкадровки для різних карт у межах [мапсету](/wiki/Glossary#beatmapset).

> Сторіборди складно робити, і на їх створення йде багато часу та знань у галузі стандартної анімації та створення графіки. osu! пропонує в [редакторі карт](/wiki/Beatmap_Editor) розділ [дизайну](/wiki/Beatmap_Editor/Design) (F2), де і відбувається створення сторібордів. Більшість завзятих творців обирають натомість програмувати безпосередньо за допомогою [скриптів](/wiki/Storyboard_Scripting). Багато хто вирішує писати програми в повнофункціональному програмуванні для створення сценаріїв, оскільки для створення складних візуальних ефектів потрібна велика кількість коду.

Reviewer: ```suggestion
Сторіборди не так і просто робити, на їх створення йде багато часу та знань у галузі стандартної анімації та створення графіки. osu! пропонує в [редакторі карт](/wiki/Beatmap_Editor) розділ [дизайну](/wiki/Beatmap_Editor/Design) (F2), де і відбувається створення сторібордів. Більшість завзятих творців обирають натомість програмувати безпосередньо за допомогою [скриптів](/wiki/Storyboard_Scripting). Багато хто вирішує писати програми в повнофункціональному програмуванні для створення сценаріїв, оскільки для створення складних візуальних ефектів потрібна велика кількість коду.
```
